docs/guides/serverless-manage-resources.ipynb

```diff
@@ -307,6 +516,66 @@
     "    tar.extractall()"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "252f0990-5643-46d4-9a58-c77fb50d17f4",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": [
+     "remove-cell"
+    ]
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Job still running, waiting 5s then retrying (19)\n"
+     ]
+    }
+   ],
+   "source": [
+    "# This cell is hidden from users.\n",
+    "# It uploads the serverless program and checks it runs.\n",
+    "\n",
+    "from qiskit.circuit.random import random_circuit\n",
+    "from qiskit_serverless import IBMServerlessClient, QiskitFunction\n",
+    "import time\n",
```

Comment:
It'd probably be good to DRY this code. Can you define a hidden helper function at the top that gets called by both?